events/views.py

```diff
@@ -37,25 +37,31 @@ class EventRegisterView(APIView):
     @swagger_auto_schema(
         operation_id="event_register",
         responses = {
-            '200': set_example(responses.event_registration_200),
+            '201': set_example(responses.event_registration_201),
+            '200':set_example(responses.user_already_registered_event_200),
             '404':set_example(responses.event_does_not_exist_404),
             '401':set_example(responses.user_unauthorized_401)
         }
     )
     def post(self,request,id):
         user = request.user
         eventregister = EventRegister()
-        if user.verified:
-            eventregister.user = user
-            try:
-                eventregister.event = Event.objects.get(id=id)
-            except:
-                return Response(responses.event_does_not_exist_404, status.HTTP_404_NOT_FOUND)    
-            else:
+        try:
+            event = Event.objects.get(id=id)
+            registeredUser = EventRegister.objects.get(user=user,event=event)
+        except:
+            if user:
```

Comment:
```suggestion
            if user.verified:
```
Check if the user is verified.

Reply:
done